src/store/CategoricalDisplayStore.js

```diff
@@ -197,6 +197,15 @@ export default class CategoricalDisplayStore {
         }, 0);
     }
 
+    @action.bound clearFilterText() {
+        this.filterText = '';
+        this.searchInput.value = '';
+        setTimeout(() => {
+            this.scroll.refresh();
```

Comment:
`this.scroll.refresh()` isn't used anymore now that we replace iScroll with perfect-scrollbar. Did you update your branch?

Reply:
so I should just use `this.updateScrollSpy()` ??

Reply:
@binarydevteam I think you should. That updates which category is highlighted. If you remove that line, chances are when you clear the filter text the active category will be in the wrong place. 

When in doubt just remove and see if it still works